components/TextAreaDetails.tsx

```diff
@@ -9,35 +9,58 @@ import { Details } from "./Details";
 import { TextArea } from "./TextArea";
 
 type Props = DynamicComponentControlledProps<string> & {
-  summary: string;
+  summary: React.ReactNode;
   name: string;
   rows?: number | null;
+  label?: {
+    id?: string | null;
+    value?: React.ReactNode | null;
+  };
+  contentAfterTextArea?: React.ReactNode;
```

Comment:
```suggestion
  contentAfterTextArea?: React.ReactNode | null;
```

Technically, `PropTypes.*` allows `null` if it allows `undefined`.